### ISSUE TITLE
How to get master token

### ISSUE THREAD

**Issue by Ashna-git-hub** [kiwiz/gkeepapi#171]
I am unable to get the master token as it does not show the sequences listed when I log in

**Comment by adrianlzt** [comment]
I followed the link I posted before. Nothing special. It just worked

**Comment by smemsh** [comment]
Yeah, with 2FA on the account, it looks like only EmbeddedSetup method works to get master token now.  It's too bad there's no script that does this, have to load a browser and look in the cookie jar :-/
